e2e-tests/desktop/tests/targets.spec.js

```diff
@@ -4,19 +4,228 @@
  */
 
 import { expect, test } from '../fixtures/baseTest.js';
+import * as boundaryHttp from '../../helpers/boundary-http';
+
+const hostName = 'Host name for test';
+let org;
+let targetWithHost;
+let targetWithTwoHosts;
+let sshTarget;
+let credential;
+
+test.beforeAll(
```

Comment:
Only created one set of resources across all tests as DC doesn't really modify the actual resources themselves. But I could still have these in a `beforeEach` or for individual tests if we don't like the sharing of resources.